lib/multibuild/build.ts

```diff
@@ -61,7 +61,7 @@ function taskHooks(
 			layers: string[],
 			fromTags: FromTagInfo[],
 		) => {
-			const tag = task.tag != null ? task.tag : imageId;
+			const tag = task.tag ?? imageId;
```

Comment:
I was afraid we might be setting empty strings on the tag, but I couldn't find any place using it actuall :shrug: